media-libs/jasper/jasper-4.0.0.ebuild

```
@@ -0,0 +1,55 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Implementation of the codec specified in the JPEG-2000 Part-1 standard"
+HOMEPAGE="https://jasper-software.github.io/jasper"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/jasper-software/jasper.git"
+else
+	SRC_URI="https://github.com/jasper-software/${PN}/archive/version-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris ~x86-solaris"
+	S="${WORKDIR}/${PN}-version-${PV}"
+fi
+
+LICENSE="JasPer2.0"
+SLOT="0"
+IUSE="doc heif jpeg opengl"
+
+RDEPEND="
+    heif? ( media-libs/libheif )
+	jpeg? ( media-libs/libjpeg-turbo:= )
+	opengl? (
+		virtual/opengl
+		>=media-libs/freeglut
```

```suggestion
		media-libs/freeglut
```
`>=` requires a version specifier